MobileDevProject/ViewController.swift

```diff
@@ -123,10 +124,22 @@ class ViewController: UIViewController, CLLocationManagerDelegate {
                 }
 
                 if isNavigating {
-                    if beacons[0].minor == navigatingBeacon.value && beacons[0].proximity == CLProximity.immediate {
+                    if beacons[0].minor == navigatingBeacon.value && beacons[0].proximity == CLProximity.near {
                         // we have arrived
                         isNavigating = false
-                        print("123223")
+                        // vibrate
+                        AudioServicesPlayAlertSound(SystemSoundID(kSystemSoundID_Vibrate))
```

Comment:
Sound system as an alert: https://developer.apple.com/documentation/audiotoolbox/1405202-audioservicesplayalertsound

And I don't think we need that. If you look here: https://github.com/tvtri96/indoor-navigation/commit/eea3e93495c4587f38418d2f88fb3d425bf5d535. We don't need AudioServicesPlayAlertSound to show the alert.